tests/functional/notification/NotificationConfigManager.js

```diff
@@ -100,13 +104,11 @@ describe('NotificationConfigManager ::', () => {
                 _id: 'example-bucket-1',
             },
         };
-        manager._metastoreChangeStream.emit('change', changeStreamEvent);
+        manager._changeStreamWrapper._changeStream.emit('change', changeStreamEvent);
```

Comment:
it would have been clearer to keep the existing name (_metastoreChangeStream), even if the type changes : it better conveys the intention that we listen to metadata (vs just knowing this is a change stream, where we need "outside" knowledge to know what we use it)

```suggestion
        manager._metastoreChangeStream._changeStream.emit('change', changeStreamEvent);
```